examples/usart.rs

```diff
@@ -1,25 +1,26 @@
 #![no_main]
 #![no_std]
 
-
 extern crate panic_halt;
 
-
 use lpc8xx_hal::{
-    prelude::*,
-    Peripherals,
-    cortex_m_rt::entry,
-    usart::BaudRate,
+    cortex_m_rt::entry, prelude::*, syscon::clocksource::PeripheralClockConfig, Peripherals,
 };
 
-
 #[entry]
 fn main() -> ! {
     let p = Peripherals::take().unwrap();
 
-    let mut swm    = p.SWM.split();
+    let mut swm = p.SWM.split();
     let mut syscon = p.SYSCON.split();
 
+    // TODO
```

Comment:
```suggestion
    // For some reason, the clock for swm needs to be enabled, even though it
    // should be enable from the start.
```

(this is the text from your comment)